surveys/sphinx-survey-2025.yml

```diff
@@ -14,81 +14,70 @@
 # People who interact with documentation, especially those who contribute to
 # or extend documentation. Use of Sphinx is encouraged but not required.
 
-title: "Sphinx Survey 2025"
+title: "Sphinx Documentation Survey (2025)"
 questions:
-- title: "Sphinx Usage Survey 2025"
+- title: "Sphinx Documentation Survey (2025)"
   type: statement
   description: >-
-    Sphinx Documentation User Survey 2025
+    We’re running this survey to understand requirements for documentation
+    tools in 2025, how they are used, and how well tools support their users.
+    This survey is being run on behalf of the [Sphinx] community, but is
+    open to anyone who works with or contributes to documentation, regardless
+    of whether you use Sphinx.
 
-    Help the Sphinx community understand who is using Sphinx, how they are
-    using it, and how well it supports the people using it. We will use this
-    information to aid community discussion about improvements to Sphinx.
+    We will use this information to aid community discussion about improvements
+    to Sphinx. Results will be public, with any identifying information removed.
+    Open text field questions may be quoted, but will not be shown in full.
 
-    Results will be public with identifying information removed. Open text
-    field questions may be quoted, but will not be shown in full.
+    This survey is run by the Sphinx community and [Quansight Labs]; survey
+    data will be deleted from any private accounts once made public. If you
+    have any questions, contact Tania Allard (Director of Quansight Labs)
+    at tallard@quansight.com and mention Sphinx Survey 2025 in the subject line.
```

Comment:
@trallard is this the best form of address for you? 

Also, we have `# Copyright (c) 2025, Quansight Inc.` at the top of this file -- would Quansight consent to licencing this work under `0BSD OR CC0-1.0` as in the wider repo, or a different licence?

Reply:
Yep that is all good.

> Also, we have # Copyright (c) 2025, Quansight Inc. at the top of this file -- would Quansight consent to licencing this work under 0BSD OR CC0-1.0 as in the wider repo, or a different licence?

This should be fine, CC0/CC-BY/0BSD, whatever folks feel is the most appropriate is also good for us.